### ISSUE TITLE
skins not applied after  updrade to 3.71

### ISSUE THREAD

**Issue by joea99** [NucleusCMS/NucleusCMS#92]
Since i cannot access the support forum looking for assistance.  Since update to 3.71 no skins work.

**Comment by yama** [comment]
https://github.com/NucleusCMS/NucleusCMS/archive/v3.8dev.zip
Thank you for the report. Could you please try this package? At this time, this package is stable. Soon, I'll release this officially.

**Comment by joea99** [comment]
Would this be an upgrade?  Should I apply it the same was as 3.71 upgrade was applied?

**Comment by yama** [comment]
Yes, it is. However, care is necessary because the folder structure is different.

1. First of all, please transfer the `_upgrades/` folder to the server.
2. Open `https://yoursite/_upgrades/` in your browser and update the DB.
3. Transfer the `nucleus/` folder, `index.php`, `action.php` to the server.

`nucleus/convert/folder/` does not need to be transferred.
We will fix it as soon as we find any problems. Please report.

**Comment by joea99** [comment]
I get the upgrade screen, login and attempt to update the DB the screen goes blank and never returns with any message.

**Comment by joea99** [comment]
Is anyone maintaining the English forum or site?   Any contacts for this?  I am attempting a fresh install on a test site with restore of database, but it has been so long I need documentation, which I cannot find.

**Comment by yama** [comment]
Sorry, I'll fix it later.
